app/main.py

```diff
@@ -1 +1,22 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command_split = command.split()
+    source_file_name = command_split[1]
+    directory = "/".join(command_split[2].split("/")[:-1])
+    destination_file_name = command_split[2].split("/")[-1]
+
+    print(source_file_name)
+    print(directory)
+    print(destination_file_name)
+
+    if directory:
+        if not os.path.exists(directory):
+            os.makedirs(directory)
+        with open(source_file_name, "r") as file, open(directory + "/" + destination_file_name, "w") as d_file:
```

Comment:
The use of `open` with two files in a single `with` statement is correct, but ensure that both files are properly closed in case of an exception. The `with` statement should handle this, but be cautious of any exceptions that might occur during file operations.